client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -24,7 +24,15 @@ const UserPage = createClass({
 	getDefaultProps : function() {
 		return {
 			username : '',
-			brews    : []
+			brews    : [],
+			sortType : 'alpha',
+			sortDir  : 'desc'
+		};
+	},
+	getInitialState : function() {
+		return {
+			sortType : 'alpha',
+			sortDir  : 'desc'
```

Comment:
The only thing here is I was surprised the default sorting was reverse-alphabetical (brews starting with Z were at the top). Can we make it "alpha", "asc" by default?

Reply:
Actually I made that the default to see if the `_.orderBy` function worked as I expected before I had created any sort of interface. So yes, it can absolutely be changed to default to 'asc'.  
Although I was thinking about whether it would be beneficial to store and load these values from localStorage, so that the UserPage remembers your last sort parameters, rather than resetting to Alphabetical every time.

Reply:
That functionality is probably better as a separate PR, though.

Reply:
I agree.